index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "format-number" {
```

Comment:
I don't think you need to wrap in `declare module` here. I think that's only when you manually include typings.